test/test_softmax.py

```diff
@@ -0,0 +1,19 @@
+import torch
+import torch.nn.functional as F
+import pytest
+from kernels.fused_softmax import triton_softmax
+
+@pytest.mark.parametrize("input_size", [(1024, 1024), (512, 512), (2048, 512)])
```

Comment:
Thanks for adding tests!

Reply:
FYI, this might not be ideal because we are not calling softmax from triton.ops like the other tests. I ran into issues with doing it that way.